nix/tenzir/plugins/source.json

```diff
@@ -2,7 +2,8 @@
   "name": "tenzir-plugins",
   "url": "git@github.com:tenzir/tenzir-plugins",
   "ref": "main",
-  "rev": "ded6b00df2d7e411b1bba149a8d510c39d61a1c4",
+  "rev": "267b65fca17d3483b9c3ba3ed5ca80b103784e6c",
   "submodules": true,
-  "shallow": true
+  "shallow": true,
+  "allRefs": true
```

Comment:
When rebasing this onto master, can I ask you to also pick up tenzir/tenzir-plugins#116 in addition to tenzir/tenzir-plugins#115?

Reply:
I simply updated to https://github.com/tenzir/tenzir-plugins/pull/116. You can keep the cherry picked commit. I'll let the current CI run finish then we should be good to merge.

Reply:
Great. I also verified manually that (1) the reconnect messages work as expected and that (2) the handshake is no longer being denied.